src/plugins/controls/public/options_list/options_list_reducers.ts

```diff
@@ -68,6 +68,7 @@ export const optionsListReducers = {
     action: PayloadAction<string>
   ) => {
     state.explicitInput.selectedOptions = [action.payload];
+    if (state.explicitInput.existsSelected) state.explicitInput.existsSelected = false;
```

Comment:
Fixes a bug where, when `singleSelect` was `true`, the `exists` option would not be deselected properly when something else was selected.